source/include/sigv4.h

```diff
@@ -44,6 +44,7 @@
  * defined in sigv4_config.h file. */
 #include "sigv4_config_defaults.h"
 
+#include "sigv4_internal.h"
```

Comment:
Why is inclusion of this required?

Reply:
It was committed mistakenly. I included it for my own testing purposes.